layouts/default.vue

```diff
@@ -1,25 +1,9 @@
 <template>
   <v-app class="app">
-    <v-app-bar elevation="0">
-      <template v-slot:prepend>
-        <router-link to="/" custom v-slot="{navigate}">
-          <div class="logo-container" @click="navigate">
-            <div class="top-bar-title">ATOR</div>
-            <img src="/images/AtorLogo.png" width="30" height="30" />
-          </div>
-        </router-link>
-      </template>
-
-      <v-app-bar-title><div class="route-title">{{ currentPageTitle }}</div></v-app-bar-title>
-
-      <template v-slot:append>
-        <ClientOnly>
-          <ConnectButton />
-        </ClientOnly>
-      </template>
-    </v-app-bar>
+    <TopBar v-if="!smallScreen" />
+    <TopBarSmallScreen v-if="smallScreen" />
 
-    <v-navigation-drawer>
+    <v-navigation-drawer v-model="navDrawer" :permanent="!smallScreen">
```

Comment:
When the nav menu is open, clicking outside of it should close the nav menu.  I think you can just use this vuetify directive for that https://vuetifyjs.com/en/directives/click-outside/